corgi/core/models.py

```diff
@@ -317,7 +326,9 @@ def save_product_taxonomy(self):
 
 
 class SoftwareBuildTag(Tag):
-    tagged_model = models.ForeignKey(SoftwareBuild, on_delete=models.CASCADE, related_name="tags")
+    tagged_model = models.ForeignKey(
+        SoftwareBuild, on_delete=models.CASCADE, related_name="tags", db_column="tagged_model_uuid"
```

Comment:
If you leave off the db_column= field, shouldn't the SoftareBuildTag's ForeignKey just default to using the related SoftwareBuild's pk?
```suggestion
        SoftwareBuild, on_delete=models.CASCADE, related_name="tags"
```

Reply:
It's the to_field attribute which does that. The db_column attribute defines the name of the foreign key column name. Since I used SQL directly to create the foreign key, I think it's better to explicitly set the column name to useto match the SQL.

Reply:
I'm fine with this approach if you think it's OK. It just makes me nervous to run migrations manually in SQL instead of letting Django generate them to handle this for us.